src/Organic/Organic.php

```diff
@@ -232,6 +233,12 @@ public function init( $apiUrl = null, $cdnUrl = null ) {
         $this->adsTxt = new AdsTxt( $this );
 
         $this->isEnabled = $this->getOption( 'organic::enabled' );
+        $this->logToSentry = $this->getOption( 'organic::log_to_sentry' );
+        // Initialize Sentry as soon as possible.
+        if ( $this->isEnabled && $this->logToSentry ) {
+            $this->initSentry( $this->sdk );
+        }
```

Comment:
Originally I initialized Sentry in `organic.php` with a default DSN to catch errors during the initialization of Organic and OrganicSDK instances even before setting a client-specific DSN. However, I don't think we can do that if we rely on an Admin Settings option.

Reply:
You can use `get_option` directly to check if we should initialize Sentry with default DSN
https://developer.wordpress.org/reference/functions/get_option/

We should skip this initialization only if `organic::log_to_sentry` is explicitly set to false

Reply:
Usually - yes, it should be done through an Organic wrapper, but here we have a special case when we need an option before we have access to the wrapper